### PR TITLE
[ENH] Distance file: Add report, Move Browse documentation data sets to button

### PR DESCRIPTION
**Pull request by janezd**

### CODE REVIEW

**Comment by lanzagar** [comment]
It is nice that we have a button to Browse documentation data sets which looks the same as in the File widget. But why exactly is this here? I cannot open iris.tab with Distance file, and unless I am missing something we have _**no**_ distance files in this directory.

**Comment by janezd** [comment]
> It is nice that we have a button to Browse documentation data sets which looks the same as in the File widget. But why exactly is this here?

Because otherwise there would be no suitable place for the report button. Seriously, this was the reason. :)

> unless I am missing something we have *no* distance files in this directory.

You were not missing anything. You were right, we had no distance files. I actually knew that, too. :)

I added two files.

File: Orange/widgets/unsupervised/owdistancefile.py [hunk 1]
```diff
@@ -81,7 +84,7 @@ def browse_file(self, in_demos=0):
             start_file = self.last_path() or os.path.expanduser("~/")
 
         filename = QtGui.QFileDialog.getOpenFileName(
-            self, 'Open Distance File', start_file)
+            self, 'Open Distance File', start_file, "(*.dst *.txt)")
```
Comment (kernc) [inline Orange/widgets/unsupervised/owdistancefile.py]: Why .txt?

Comment (janezd) [inline Orange/widgets/unsupervised/owdistancefile.py]: I thought I had some .txt files with distances, so I assumed this used to be a valid extension and wanted to support it. Stupid. I removed it.

File: Orange/datasets/slovenian-towns.dst [hunk 2]
```diff
@@ -0,0 +1,92 @@
...
+Škofja Loka	0.50387	0.69322	1.99757	0.28284	0.36893	0.76830	1.19745	1.32707	1.02862	0.96896	0.52967	0.37268	0.62650	1.06784	0.30185	0.84344	0.61464	1.77678	0.53359	2.13983	0.32872	0.60208	0.89567	0.36705	0.32059	0.67185	0.72150	0.77817	0.66604	0.83749	0.09718	0.60599	1.20012	0.24608	0.93348	1.58894	0.54594	1.93197	0.73786	0.50607	1.40337	1.14042	0.73352	0.98036	0.46338	0.56691	0.95510	1.93247	0.97653	0.59114	0.68516	0.94104	1.86682	1.44962	0.96896	0.71259	0.64313	0.95525	0.39229	1.38373	1.58649	0.88898	1.81115	1.02116	0.21731	0.67185	0.61283	1.33500	1.04137	1.10114	0.63683	0.78102
+Slovenjgradec	1.35688	1.53912	1.16964	0.99471	1.15710	1.54425	0.66667	0.80450	1.84037	0.33747	1.38122	1.01352	0.23863	0.95525	0.61689	0.08333	1.45774	0.93169	0.34359	1.28949	1.18521	1.26370	1.72450	1.01885	0.55478	1.51337	1.52352	0.90952	1.52069	1.66040	0.78031	1.30043	0.68007	0.73409	0.39616	0.75296	0.52941	1.11667	1.48791	1.36178	0.56765	0.89753	1.59103	0.15811	1.15193	1.39891	1.72127	1.09367	0.13017	1.33998	1.54128	0.72150	1.07134	0.60576	1.80755	1.56009	1.33344	1.78893	1.13529	0.59838	0.78969	0.46098	0.97368	0.16667	0.91530	1.36677	0.85797	0.61869	1.88164	0.43621	1.46534	1.59670	0.85797
```
Comment (astaric) [inline Orange/datasets/slovenian-towns.dst]: Slovenj Gradec?

File: Orange/datasets/slovenian-towns.dst [hunk 3]
```diff
@@ -0,0 +1,92 @@
...
+Nova Gorica	0.24267	0.10541	2.68002	0.61328	0.44845	0.39616	1.83364	1.95064	0.56691	1.64131	0.16667	0.73220	1.30650	1.58052	0.96753	1.52142	0.25927	2.46176	1.21804	2.82499	0.37006	0.71200	0.41700	0.65085	1.00277	0.13437	0.30732	1.25720	0.15723	0.40859	0.76158	0.54975	1.83341	0.87242	1.59591	2.27334	1.18755	2.61220	0.45277	0.17951	2.08806	1.69345	0.06009	1.66550	0.57470	0.21667	0.64334	2.61672	1.65672	0.44472
+Novo Mesto	1.28604	1.57003	1.34670	1.20784	1.30916	1.70237	0.36591	0.44472	1.58894	0.44472	1.40574	0.80000	0.73805	0.23393	0.65744	0.79460	1.33375	1.21438	0.61033	1.55849	1.16726	0.94590	1.52352	1.27769	0.69001	1.55929	1.64604	0.34319	1.43343	1.45497	0.93244	1.54290	0.35785	0.69642	0.35629	1.02862	0.41667	1.25753	1.67763	1.36728	0.89225	0.21213	1.55322	0.66999	1.39413	1.47545	1.41264	1.32330	0.81667	1.53206	1.52407
```
Comment (astaric) [inline Orange/datasets/slovenian-towns.dst]: Novo mesto?

**Comment by lanzagar** [comment]
Much better now with some actual documentation data to browse :)
Just make sure the added .tab file is added to the datasets dict so the test does not fail (you have to run `list_update.py` in the datasets dir)

**Pull request merged**